src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageLoggerExtensions.cs

```diff
@@ -122,5 +128,13 @@ public static void MalformedPageDirective(this ILogger logger, string filePath,
                 _malformedPageDirective(logger, filePath, messages, null);
             }
         }
+
+        public static void UnsupportedAreaPath(this ILogger logger, RazorPagesOptions options, string filePath)
+        {
+            if (logger.IsEnabled(LogLevel.Warning))
+            {
+                _unsupportedAreaPath(logger, filePath, options.AreaRootDirectory, options.AreaRootDirectory, options.RootDirectory, null);
+    }
+}
```

Comment:
I think the diff got confused here, if not. Indentation?_

Reply:
Probably a bad merge. Will fix.